pkg/packages/manager.go

```diff
@@ -150,7 +150,9 @@ func processInstalling(mc *ManagerContext) bool {
 		mc.Log.Error(err, "Initialization failed")
 		return true
 	}
-	if err := mc.PackageDriver.Install(mc.Ctx, mc.Package.Name, mc.Package.Spec.TargetNamespace, mc.Source, values); err != nil {
+
+	createNamespace := mc.PBC.Spec.CreateNamespace
```

Comment:
variable not necessary here?

Reply:
No it's not necessary per-se but I prefer that over a function call with a bunch of deep property access.